src/store/actions/contactInfo.js

```diff
@@ -1,9 +1,23 @@
 import { ACTION_TYPES } from '../actionTypes';
 
+const dataSanitization = data => {
+  if (data) {
+    return data.map(contact => {
+      return {
+        _id: contact._id,
+        title: contact.title,
+        url: contact.url,
+        sub_title: contact.sub_title,
```

Comment:
I'm noticing that these fields are using snake_case naming convention, maybe you should use camelCase instead, to stay consistent with the general conventions of the codebase, example: use `subTitle` instead of `sub_title`